DEVX-1942: Create first producer application (#491)
confluentinc/kafka-tutorials: _includes/tutorials/kafka-producer-application/kafka/markup/dev/run-consumer.adoc

////
  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
  of the tutorial.  Change the text as needed.

////

> Now let's run a console consumer on the output topic to confirm you're application parsed the file correctly and published the expected records.

Reviewer: ```suggestion
Now run a console consumer that will read topics from the output topic to confirm your application published the expected records.
```
